streamparse/thrift.py

```diff
@@ -0,0 +1,569 @@
+"""
+Python Storm Topology DSL
+"""
+
+import io
+
+import six
+import thriftpy
+
+# Apache storm.thrift file as string to simplify loading it.
+_THRIFT_STR = """
```

Comment:
Mostly because I wanted to make sure things worked from inside an egg/zip.  I know there's a way to access data files even then, but I haven't done it before.